contracts/token/SoulboundERC721.sol

```diff
@@ -33,10 +34,15 @@ contract SoulboundERC721 is ERC721Upgradeable, ERC721EnumerableUpgradeable, IERC
     ) public view virtual override(ERC721EnumerableUpgradeable, ERC721Upgradeable, IERC165Upgradeable) returns (bool) {
         return
             interfaceId == 0x49064906 || // ERC4906
+            interfaceId == type(IERC5192).interfaceId ||
             interfaceId == type(IERC721EnumerableUpgradeable).interfaceId ||
             super.supportsInterface(interfaceId);
     }
 
+    function locked(uint256) external pure returns (bool) {
```

Comment:
According to the spec, for all tokens that don‘t exist („all tokens assigned to the zero address“) the locked function must throw.


Anyways, there also exist implementations where we forgot this - but maybe it‘s good if we strictly go by the spec.

Reply:
Fair enough. Implemented in 357e8df